pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
```

Comment:
This is probably the second most interesting thing – migration from `setup.py` to `pyproject.toml`.